pkg/crds/controllers/batch/batchjob_controller_helpers.go

```diff
@@ -70,6 +71,22 @@ type batchJobStatusInfo struct {
 	TotalBatchCount int
 }
 
+func (r *BatchJobReconciler) checkIfConfigMapExists(ctx context.Context, batchJob batch.BatchJob) (bool, error) {
```

Comment:
this function can be renamed to`getConfigMap` and return the config map instead of a boolean, since we need information about the config map later on. This would be similar to what is done with the worker job

Reply:
Sure that would work.